### PR TITLE
Update tiuri.json

### PR DESCRIPTION
**Pull request by ituir**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/tiuri.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
   },
   "record": {
     "A": [
-      "138.201.76.10"
+      "212.227.129.140"
```
Comment (WilliamDavidHarrison) [inline domains/tiuri.json:10]: Please upload your website to the server.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/tiuri.json (hunk 1 above)
Comment (ituir) [reply domains/tiuri.json:10]: I did, and I changed from server. You cant see the website when you visit the ip because it works with nginx virtualhosts.

Comment (ituir) [reply domains/tiuri.json:10]: But I Will bring it online on a different domain So you can see the website

Comment (WilliamDavidHarrison) [reply domains/tiuri.json:10]: @ituir Please send the domain

**Comment by ituir** [comment]
dev.tiuri.be

File: domains/tiuri.json (hunk 1 above)
Comment (WilliamDavidHarrison) [reply domains/tiuri.json:10]: Is this it?
Screenshot_20221121_153524_Chrome

Comment (ituir) [reply domains/tiuri.json:10]: yes, it is not very complicated!

Comment (WilliamDavidHarrison) [reply domains/tiuri.json:10]: > yes, it is not very complicated!

Sorry, you can't use that as your website.

**Review by wdhdev — APPROVED** [review]

**Review by andrewstech — APPROVED** [review]

**Pull request merged**